### ISSUE TITLE
Devtools Form Floating Can't Debug HTML Pages

### ISSUE THREAD

**Issue by lidingkun** [salvadordf/CEF4Delphi#217]
Demand:
The requirement is to implement Devtools floating on projects using CEF, and to debug related HTML pages using devtools.Like the first button of chrome devtool Dock side. There is also a problem with demo testing shared on CEF official website. CEF version: embedded chromium version: 75.0.3770, demo path: 'CEF4Delphi-master\demos\Delphi_VCL\MiniBrowser\'

Internal implementation of the code:
The 'TChromium.ShowDevTools' procedure in uCEFChromium. pas file initializes the Devtools form. 'InitializeDevToolsWindowsInfo' uses the 'WindowsInfoAsChild' procedure or 'WindowsInfoAsPopUp' in uCEFMiscFunctions. pas file to implement the form embedded in chromium or floating on chromium.

code in Delphi demo:
   procedure TChromium.InitializeDevToolsWindowInfo(aDevTools : TWinControl);
begin
  {$IFDEF MSWINDOWS}
  if (aDevTools <> nil) then
    WindowInfoAsChild(FDevWindowInfo, aDevTools.Handle, aDevTools.ClientRect, aDevTools.Name)
   else
    WindowInfoAsPopUp(FDevWindowInfo, WindowHandle, DEVTOOLS_WINDOWNAME);     
  {$ELSE}
  if (aDevTools <> nil) then
    WindowInfoAsChild(FDevWindowInfo, aDevTools.Handle, aDevTools.ClientRect)
   else
    WindowInfoAsPopUp(FDevWindowInfo, WindowHandle);
  {$ENDIF}
end;

By the way, this is the first time I have used this website to give feedback. As a software developer in China, I hope you can understand my English.

**Comment by salvadordf** [comment]
If you need to use a popup window for the DevTools you can call TChromium.ShowDevTools with a NIL as the second parameter.


In the MiniBrowser demo you would only need to modify TMiniBrowserFrm.ShowDevTools in uMiniBrowser.pas like this :
`procedure TMiniBrowserFrm.ShowDevTools(aPoint : TPoint);
begin
  Chromium1.ShowDevTools(aPoint, nil);
end;`

**Comment by lidingkun** [comment]
Maybe you didn't understand what I meant. I have used "Chromium1.ShowDevTools(aPoint, nil);" in the MiniBrowser demo. The problem is on this. When MiniBrowser.exe is running, use F12 to open devtools. At this time, although the window pops up, it is impossible to debug HTML. For example, the size of a font in the page cannot be changed, the cursor will focus on the parent page, and the data in the styles form on the right side of devtools cannot be modified.
However, I just found out the reason for this situation. I have defined a "ShowDevTools" process to cause this problem. I think I know how to solve it.
Finally, thank you very much for answering the question I asked before.
